src/sentry/rules/filters/issue_occurrences.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import
+
+from django import forms
+
+from sentry.rules.filters.base import EventFilter
+
+
+class IssueOccurrencesForm(forms.Form):
+    value = forms.IntegerField()
+
+
+class IssueOccurrencesFilter(EventFilter):
+    form_cls = IssueOccurrencesForm
+    form_fields = {
+        "value": {"type": "number", "placeholder": 10},
+    }
+
+    label = "An issue has happened at least {value} times"
+
+    def passes(self, event, state):
+        try:
+            value = int(self.get_option("value"))
+        except (TypeError, ValueError):
+            return False
+
+        issue_occurrences = event.group.times_seen
```

Comment:
Could you add a comment somewhere around this filter just noting that this filter might not always work exactly when we cross over the value, due to us batching writes to `times_seen`?